app/views/__tests__/ExportCodeInput.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  NavigationScreenProp,
+  NavigationRoute,
+  NavigationState,
+  NavigationParams,
+} from 'react-navigation';
+import { render, cleanup } from '@testing-library/react-native';
+import '@testing-library/jest-native/extend-expect';
+import { fireEvent } from '@testing-library/react-native';
+import { TracingStrategy } from '../../../app/tracingStrategy';
```

Comment:
I might group these imports such that third party code is separated from our code by a line break